sc/social/like/locales/pt_BR/LC_MESSAGES/sc.social.like.po

```diff
@@ -422,3 +422,12 @@ msgstr "Cancelar"
 #: sc/social/like/vocabularies.py:36
 msgid "vertical"
 msgstr "vertical"
+
+
```

Comment:
Not the end of the world, but this line can be deleted. Are you using `bin/rebuild_i18n-sh` to manipulate your `*.po*?